src/content/reference/react/use.md

```diff
@@ -269,13 +290,130 @@ export function fetchMessage() {
 
 <Solution />
 
-</Recipes>
+#### Conditional data fetching {/*conditional-data-fetching*/}
+
+In this example, the `Article` component only calls `use` when the `shouldIncludeByline` is set to true. If `shouldIncludeByline` is set to true, the Promise returned by `fetchByline()` is read by the use Hook. The value read by `use` Hook is then used to display the byline of the article's author to the user.
+
+<Sandpack>
+
+```js App.js
+'use client';
+
+import { use, Suspense, useState } from 'react';
+import { fetchByline } from './api.js';
+
+function Article({ title, body, shouldIncludeByline, authorId }) {
+  const [byline, setByline] = useState(null);
+
+  if (shouldIncludeByline && !byline) {
+    // Because `use` is inside a conditional block, we avoid blocking
+    // unnecessarily when `shouldIncludeByline` is false.
+    setByline(use(fetchByline(authorId)));
+  }
+
+  return (
+    <div>
+      <h1>{title}</h1>
+      <h2>{byline}</h2>
+      <section>{body}</section>
+    </div>
+  );
+}
+
+export default function App() {
+  const [show, setShow] = useState(false);
+  function showArticle() {
+    setShow(true);
+  }
+
+  if (!show) {
+    return <button onClick={showArticle}>Show article</button>;
+  }
+  return (
+    <Suspense fallback={<p>⌛Loading article...</p>}>
+      <Article
+        title={'Article Title'}
```

Comment:
Is the docs code style to write this as `title="Article Title"` or `title={'Article Title'}`. I'm used to seeing the former

Reply:
Looks like the docs style is `title="Article Title"`! Fixed.